tests/tutorial-test/cases/accordionHints.md

```diff
@@ -0,0 +1,10 @@
+# Accordion Hints
+## Step 1
+Welcome!
+
+~hint This content is hidden until the user clicks here.
```

Comment:
Some more tests with the multiple hints and nested hints would be great. If we are not supporting nested hints we should call out (I am ok not supporting nested hints as well.

Reply:
> Some more tests with the multiple hints and nested hints would be great. If we are not supporting nested hints we should call out (I am ok not supporting nested hints as well.

I'm perfectly fine not supporting nested hints right now…that said, if we can get them, I'm sure they'll get used!  They'd just have to have slightly differentiated CSS so people could tell what's going on.

Reply:
I will add more test cases and keep nested accordion hints as not supported. I'll add a notice in the docs.